docs/graphql-java-comparison.md

```diff
@@ -0,0 +1,146 @@
+---
+id: graphql-java-comparison
+title: GraphQL Java Comparison
+---
+
+[graphql-java](https://graphql-java.com/) is one of the more popular JVM based GraphQL implemenations. GraphQL Kotlin is
+built on top of `grahpql-java` as there are many parts of the GraphQL specification that are trivial and this implemenatation
+has been used and tested by many users.
+
+### GraphQL Java Schema
+
+The most common way to create the schema in `graphql-java` is to first manually write the SDL file:
+
+```graphql
+schema {
+    query: Query
+}
+
+type Query {
+    bookById(id: ID): Book
+}
+
+type Book {
+    id: ID!
+    name: String!
+    pageCount: Int!
+    author: Author
+}
+
+type Author {
+    id: ID!
+    firstName: String!
+    lastName: String!
+}
+```
+
+Then write the runtime code that matches this schema to build the `GraphQLSchema` object.
+
+```java
```

Comment:
do we want to change this example to Kotlin (so it is easier to compare vs plain java)?

Reply:
Showing it in java makes it look more verbose intentionally 😜

Reply:
This comes from this example: https://www.graphql-java.com/tutorials/getting-started-with-spring-boot/